hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/om/FSORepairTool.java

```diff
@@ -418,8 +404,7 @@ private void handleUnreachableObjects(OmVolumeArgs volume, OmBucketInfo bucket)
           // NOTE: We are deserializing the proto of every reachable file
           // just to log it's size. If we don't need this information we could
           // save time by skipping this step.
-          reachableBytes += fileInfo.getDataSize();
-          reachableFiles++;
+          reachableStats.add(new ReportStatistics(0, 1, fileInfo.getDataSize()));
```

Comment:
Let's avoid object churn by introducing with dir- and file-specific methods in `ReportStatistics`:

```java
  public void addDir() {
    dirs++;
  }

  public void addFile(long size) {
    files++;
    bytes += size;
  }
```